scripts/build/release.sh

```diff
@@ -153,6 +153,16 @@ mv /tmp/version.h              src/app/contextBroker/version.h
 rm -rf CHANGES_NEXT_RELEASE
 touch CHANGES_NEXT_RELEASE
 
+# Adjust Readthedocs documentation badge. Note that the procedure is not symetric (like in version.h), as
```

Comment:
Typo fixed in last commit (sorry, at mobile it is difficult to paste hash).

Regarding readthedocs, if @jmcanterafonseca hasn't raised comment I think is ok.